src/Raven.Client/ServerWide/Tcp/TcpConnectionHeader.cs

```diff
@@ -16,5 +18,15 @@ public enum OperationTypes
         public string SourceNodeTag { get; set; }
 
         public OperationTypes Operation { get; set; }
+
+        public int OperationVersion { get; set; }
+
+        public static readonly Dictionary<OperationTypes, int> TcpVersions = new Dictionary<OperationTypes, int>
+        {
```

Comment:
I think that it betters to use a different version number here, to be sure that we aren't mixing them.